Sources/MapboxCoreNavigation/RoutingProvider.swift

```diff
@@ -8,13 +8,26 @@ import MapboxDirections
  */
 public protocol RoutingProvider {
     
+    typealias IndexedRouteResponseCompletionHandler = (_ result: Result<IndexedRouteResponse, DirectionsError>) -> Void
+    
+    /**
+     Routing caluclation method.
```

Comment:
[MapboxDirections](https://github.com/mapbox/mapbox-directions-swift/blob/3e0d1ac37aca0489965028e4521623531a02cb7d/Sources/MapboxDirections/Directions.swift#L168) has a more descriptive comment for a similar method. (×2)

```suggestion
     Begins asynchronously calculating routes using the given options and delivers the results to a closure.
```